src/v2/Components/AuctionResultPerformance.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import {
+  Text,
+  ThemeProviderV3,
+  IncreaseIcon,
+  Flex,
+  DecreaseIcon,
+  Spacer,
+} from "@artsy/palette"
+
+export const AuctionResultPerformance = ({
+  value,
+  align,
+}: {
+  value: string | null
+  align: "left" | "right"
+}) => {
+  if (value === null) {
+    return null
+  }
+
+  const sign = value[0] === "-" ? "down" : "up"
+  const color = sign === "up" ? "green100" : "red100"
+  const text = sign === "up" ? value : value.slice(1)
+  const Arrow = sign === "up" ? IncreaseIcon : DecreaseIcon
+
+  return (
+    <ThemeProviderV3>
+      <Flex
+        flexDirection="row"
+        justifyContent={align === "right" ? "flex-end" : undefined}
+        alignItems="baseline"
+      >
+        <Arrow fill={color} width={10} height={10} />
+        <Spacer mr="3px" />
+        <Text variant="xs" fontWeight={400} color={color}>
```

Comment:
@dzucconi - is this the proper way to do bold type? (I've been meaning to ask)

Reply:
Just `fontWeight="bold"`. We only have the one weight.

Reply:
I'll ask Jonathan. figma used to have more info but not this time.